### PR TITLE
added existence check before looping on array

### PR DESCRIPTION
**Pull request by karimation**
if `pageEntry.messaging` doesn't exist, the old code throws an error, so i added a check before using that variable.

### CODE REVIEW

File: gui-webview/routes/webhooks.js [hunk 1]
```diff
@@ -60,19 +60,21 @@ router.post('/', (req, res) => {
...
-        } if (messagingEvent.postback) {
-          receiveApi.handleReceivePostback(messagingEvent);
-        } else {
-          console.log(
-            'Webhook received unknown messagingEvent: ',
-            messagingEvent
-          );
-        }
-      });
+      if(pageEntry.messaging){
```
Comment (tcsapunaru) [inline gui-webview/routes/webhooks.js]: Can you do 

`if (!pageEntry.messaging) {
  return;
}`

and leave the original code in place? The benefit is easier reading. Also a <space> in between if and (.

BTW this was done as it is since the bot processes only messaging events. But you are right no harm in having the extra check.

Comment (karimation) [inline gui-webview/routes/webhooks.js]: yes, you can use  `if (!pageEntry.messaging) { return; }`
and i think your alternative has the same effect with cleaner code.

Comment (karimation) [inline gui-webview/routes/webhooks.js]: will you fix it or should i fix it and make another pull request ?

**Comment by tcsapunaru** [comment]
Fixed here --> https://github.com/fbsamples/messenger-bot-samples/commit/fb8332e954c8ee59061319850db18886e0062508
